source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -22,37 +22,36 @@ Before you begin, go to your SendGrid [account settings]({{site.app_url}}/user/a
 Adding a Sender
 {% endanchor h2 %}
 
-1. Click **Create New Sender**
-1. Fill out the entire "Add Sender Identity" form. This form is so extensive so that you are CAN-SPAM compliant. The footer of your emails includes the information you provide here.
+This is to ensure that you are CAN-SPAM compliant. The footer of your emails includes the information you provide here.
 
-   * **From Name** - This is user-friendly name that is displayed to the user when they receive their email.
-   * **From Email Address** - This will display to the user as the email address who sent this email.
-   * **Reply-To Email Address** - If your user hits reply in their email, the reply will go to this address.
-   * **Company Address, City, State, Zip Code, Country** - The address of your business, for CAN-SPAM compliance.
-   * **Nickname** - This is the name of this sender identity, which can be useful for identifying this identity in your list of senders. It will not be visible to your recipients.
```

Comment:
this seems repetitive, how about "A label for your sender identity to help you identify it more quickly - it is not visible to your recipients."